be1-go/channel/coin/uint53/mod.go

```diff
@@ -0,0 +1,37 @@
+/*
+  Operations on 53-bits non-negative integers.
+
+  These values can be handled without loss of precision in IEEE754 64-bit
+  floating numbers, and thus in JSON.
+*/
+package uint53
+
+import (
+	"errors"
+)
+
+// Internal representation in this implementation
```

Comment:
Comments on public types and functions must "...start with complete sentences naming the declared symbol".
We always start the comment with the symbol's name:

```go
// Uint53 defines the internal representation in this implementation
```

Same bellow.